hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncBatchRpcRetryingCaller.java

```diff
@@ -487,6 +487,10 @@ private void tryResubmit(Stream<Action> actions, int tries, boolean immediately,
     } else {
       delayNs = getPauseTime(pauseNsToUse, tries - 1);
     }
+    Optional<MetricsConnection> metrics = conn.getConnectionMetrics();
```

Comment:
Move this inside the below if condition block?

Reply:
Thanks, done!